presto-product-tests/bin/run_on_docker.sh

```diff
@@ -167,6 +168,16 @@ if [[ "$ENVIRONMENT" == "multinode" ]]; then
    PRESTO_SERVICES="${PRESTO_SERVICES} presto-worker"
 fi
 
+CLI_ARGUMENTS="--server presto-master:8080"
```

Comment:
Generally you can't export variables to the parent process. `${DOCKER_CONF_LOCATION}/${ENVIRONMENT}/compose.sh` is being run in a new shell (process). 

```
⌘ printf "#!/bin/bash\nexport MY_VARIABLE='myvalue'" > script.sh && chmod 700 script.sh
⌘ ./script.sh 
⌘ echo ${MY_VARIABLE}

⌘ source ./script.sh 
⌘ echo ${MY_VARIABLE}
myvalue

```

From other side we can include the `compose.sh` script to the current shell, so all the variables will be exported. But we need to send some of the `environment_compose` processes to the background (https://github.com/Teradata/presto/blob/43618af0fe032646cd21a023b5460229ac4cdd7f/presto-product-tests/bin/run_on_docker.sh#L204), and we are passing arguments to that script. When you run `source some_script.sh` - the arguments of the current shell are being propagated to that script as part of the context of the current shell.